### DISCUSSION TITLE
Learning Resources

### DISCUSSION THREAD

**Discussion by dominicduffin1** [Virtual-Coffee/virtualcoffee.io#1123] [Monthly Challenge]
## Learning Resources

The aim of this discussion is to collect good resources in one place as a VC Recommended Resources Guide. This doubles as a place to collect, share and find resources for the Month of Learning challenge.

Please only post resources that you have personally used and recommend or that you know from reliable sources that it is a good resource.

Please use the format below if possible:

---

### Tech Category - specific language
[Resource Title](https://github.com/orgs/Virtual-Coffee/discussions/link) - **type of resource and author** - short description or main topic - **COST:** (if applicable)

For example:

### Frontend - React/Gatsby
[Accessibility in JavaScript Applications](https://frontendmasters.com/courses/javascript-accessibility/) - **Frontend Masters Course - Marcy Sutton** - We can produce innovative and inclusive JavaScript-powered web apps! This course will teach you to remove barriers to access that might prevent people with disabilities from using a modern JavaScript web application. We’ll study accessibility in UI components, primarily with React and Gatsby.js, but with knowledge applicable to all JavaScript-heavy web stacks. Debug your sites for accessibility, manage keyboard focus, announce live updates to the page to screen readers, and use manual and automated testing to gain web accessibility superpowers! - **COST:** Frontend Masters Subscription at $39/month with options for yearly and team.

---

[Resources from 2022](https://github.com/orgs/Virtual-Coffee/discussions/496)

**Comment by meg-gutshall** [comment]
I started a similar thread from the Rust Learning Cohort over here: https://github.com/orgs/Virtual-Coffee/discussions/1122#discussioncomment-8429375. I put a key with little emoji codes as well. Take a look and see if you want to adopt it!

**Comment by meg-gutshall** [comment]
What may actually be even better is collecting everything in a repo. Since we can create [Pull Request Templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository), maybe we can make something simple for members to use when they want to share resources. I'll noodle on it and come back here when I've had a chance to experiment a little.

**Comment by BekahHW** [comment]
Ideally I’d like all of these resources to live on the site. I’m working through a process now to add these to a site as well as tiers (discussion v. On our site). We’ve tried having resources in other repos and that’s never taken. The goal for resources is to support current VC events and common questions on our site. On Apr 13, 2024, at 5:27 PM, Meg Gutshall ***@***.***> wrote:﻿
What may actually be even better is collecting everything in a repo. Since we can create Pull Request Templates, maybe we can make something simple for members to use when they want to share resources. I'll noodle on it and come back here when I've had a chance to experiment a little.
